docs/4.0/quickstart.md

```diff
@@ -163,6 +163,31 @@ new-node      xxxxx-xxxx-xxxx-xxxxxxx     10.0.10.2:3022
 !!! tip "NOTE": 
     Teleport also supports static pre-defined invitation tokens which can be set in the [configuration file](admin-guide.md#adding-nodes-to-the-cluster)
 
+## Adding a node located behind NAT
+With the current setup you've only been able to add nodes that have direct access to the 
+authentication server and within the internal IP range of the cluster. We recommend 
```

Comment:
I think everywhere else we call it just "auth server", whatever we use, we should be consistent in the docs. Same for below.